### DISCUSSION TITLE
wie kann ich auf eine bestimmte Trunk-Version auschecken?

### DISCUSSION THREAD

**Discussion by muckel-sassi** [Freetz-NG/freetz-ng#209] [Allgemein] [answered]
Moin,
wie oben beschrieben bräuchte ich die Info, wie ich auf eine bestimmt Version auschecken kann.

z.B. auf die version von Changeset 17979

**Comment by Conan179** [comment, marked as answer]
Steht auf der Hauptseite ;)
Checkout old revision:
  git checkout HASH-OF-COMMIT # -b NEW-BRANCH

**Comment by muckel-sassi** [comment]
Ah, ok, vielen Dank.
Das steht da ber noch nicht so lange, oder?

**Comment by Conan179** [comment]
Nur seit dem 25 Nov 2020 https://github.com/Freetz-NG/freetz-ng/commit/9adcfa56c2cfeeb69b08a41e9456d5657207a3d2

**Comment by muckel-sassi** [comment]
ups, wie die Zeit vergeht... ok, geb mich geschlagen... ;)
